src/components/FindWallet/WalletFilterPersona.tsx

```diff
@@ -0,0 +1,149 @@
+              border="solid"
+              borderColor={getPersonaBorderColor(selectedPersona, idx)}
+              bg="background.highlight"
+              borderRadius="base"
+              cursor="pointer"
+              transition="0.5s all"
+              _hover={{
+                borderColor: "primary.hover",
+              }}
+              role="group"
+              onClick={() => {
+                if (idx === selectedPersona) {
+                  resetFilters()
+                  trackCustomEvent({
+                    eventCategory: "UserPersona",
+                    eventAction: `${persona.title}`,
+                    eventName: `${persona.title} false`,
+                  })
+                } else {
+                  setSelectedPersona(idx)
+                  setFilters(persona.presetFilters)
+                  trackCustomEvent({
+                    eventCategory: "UserPersona",
+                    eventAction: `${persona.title}`,
+                    eventName: `${persona.title} true`,
+                  })
+                }
+              }}
+            >
+              <Flex gap={2} mb={showMobileSidebar ? 0 : 1} px={1.5}>
+                <Box role="radio" aria-label={`${persona.title} filter`}>
+                  <Icon
+                    as={MdCircle}
+                    borderRadius="full"
+                    boxSize={2.5}
+                    my={0}
+                    mx={1}
+                    fill={
+                      selectedPersona === idx
+                        ? "primary.base"
+                        : "rgba(0, 0, 0, 0)"
+                    }
+                    background={
+                      selectedPersona === idx ? "primary.base" : "transparent"
+                    }
+                    outline="1.5px solid"
+                    outlineColor="primary.base"
+                    outlineOffset="3px"
+                    fontSize={8}
+                    _groupHover={{ outlineColor: "primary.hover" }}
+                  />
+                </Box>
+
+                <Heading
+                  as="h3"
+                  ms={0}
+                  fontSize={showMobileSidebar ? "md" : "xl"}
+                  fontWeight="bold"
+                  py={0}
+                  pe={1}
+                  ps={0}
+                  lineHeight="1.7rem"
+                  color="primary.base"
+                  _groupHover={{
+                    color: "primary.hover",
+                  }}
+                >
+                  {persona.title}
+                </Heading>
+              </Flex>
+
+              <Text
+                display={showMobileSidebar ? "none" : "block"}
+                p="0.4rem"
+                color={selectedPersona === idx ? "text" : "text200"}
+                fontSize="sm"
+                fontWeight="normal"
+                transition="0.5s all"
+                lineHeight={1.3}
+              >
+                {persona.description}
+              </Text>
```

Comment:
Just noting, I feel like we use the `{someBoolean && <ConditionalComponent {...} />}` pattern much more than `display{someBoolean ? "none" : "block"}` pattern. Not sure if we have this documented anywhere... not a blocker but just noting

Also, maybe a question for @nloureiro, but is it possible to move away from custom padding sizes here? (ie. `0.4rem`, or `6.4px`)... could we not just use `"2"` here for half a rem?